ext/standard/string.c

```diff
@@ -3989,18 +3989,22 @@ static zend_always_inline char *php_stripslashes_impl(const char *str, char *out
 		vst1q_u8(q.mem, vceqq_u8(x, vdupq_n_u8('\\')));
 		if (q.dw[0] | q.dw[1]) {
 			int i = 0;
```

Comment:
Minor nit while you are at it, do you mind changing this to an unsigned int, as this would prevent some implicit signed to unsigned conversions.